src/databricks/labs/blueprint/paths.py

```diff
@@ -448,17 +441,296 @@ def match(self, path_pattern, *, case_sensitive=None):
                 return False
         return True
 
-    def as_fuse(self):
+    def home(self):  # pylint: disable=arguments-differ
+        """Return the user's home directory. Adapted from pathlib.Path"""
+        return type(self)(self._ws, "~").expanduser()
+
+    @abstractmethod
```

Comment:
please pull all abstract methods up in the body of the class, that's where they're logically expected